Model/Product/Builder.php

```diff
@@ -240,6 +240,50 @@ public function build(
         return $nostoProduct;
     }
 
+    /**
+     * Adds selected attributes to all tags also in the custom fields section
+     *
+     * @param Product $product
+     * @param Store $store
+     * @return array
+     */
+    private function amendAttributesCustomFields(Product $product, Store $store)
```

Comment:
Change the method name. Because it is a method returning data, but not modifying the product inside this method.